lib/ssh/doc/src/ssh.xml

```diff
@@ -1107,6 +1107,88 @@
      </desc>
    </func>
 
+<!-- direct_tcpip/5 -->
+   <func>
+     <name name="direct_tcpip" arity="5" since=""/>
+     <fsummary>Forward local port to a server.</fsummary>
+     <desc>
+       <p>Listen on <c>LocalHost</c>:<c>LocalPort</c> and forward connections to <c>RemoteHost</c>:<c>RemotePort</c>. 
+       </p>
+       <p>Returns a port assigned on the client.</p>
+       <p> <c>LocalHost</c> interpretation:</p>
+       <list>
+         <item><c>"0.0.0.0"</c> means to listen on all IPv4 addresses</item>
+         <item><c>"::"</c> means to listen on all IPv6 addresses</item>
+         <item><c>loopback</c> means to listen on loopback interface</item>
+         <item><c>"127.0.0.1"</c> and <c>"::1"</c> indicate listening on the loopback
+         interfaces for IPv4 and IPv6, respectively</item>
+         <item>other values are interpreted as usually</item>
+       </list>
+       <p>If <c>LocalPort</c> is 0, a listener is assigned to an arbitrary free port</p>
+     </desc>
+   </func>
+<!-- tcpip_forward/5,6 -->
+   <func>
+     <name since="">tcpip_forward(ConnectionRef, RemoteHost, RemotePort, LocalHost, LocalPort) -> Result</name>
```

Comment:
We have an ongoing task to re-write the SSH documentation using the -spec in the source to generate the documentation like you have used for ssh:direct_tcpip/5.  Please use that style also for tcpip_forward/5,6

Reply:
Thank you,
fixed